packages/tees/__test__/src/lifecycle/postSetup.test.js

```diff
@@ -0,0 +1,230 @@
+            expect(global.__afterEachCase__).toEqual(result.afterEachCase);
+
+            expect(global.beforeEachStart).toBeCalled();
+            expect(global.beforeEachStart).toHaveBeenCalledWith(result.context, result.beforeEachCase);
+
+            expect(result.context.driver.run).toBeCalled();
+
+        });
+
+        it('execCase, if test only should assign the params of global', async () => {
+
+            context.isOnly = true;
+
+            result = await getExecCaseParams(context);
+            global.getExecCaseParams = jest.fn().mockReturnValue(result);
+
+            global.beforeEachStart = jest.fn();
+
+            result.context.driver.run = jest.fn();
+            result.context.driver.newPage = jest.fn();
+            result.context.driver.goto = jest.fn();
+
+            await execCase(context);
+
+            expect(result.context.driver.run).toBeCalled();
+        });
+
+        it('execCase, if isUT is true context.driver.run should not be called', async () => {
+
+            context.driver = "ut";
+
+            result = await getExecCaseParams(context);
+
+            global.getExecCaseParams = jest.fn().mockReturnValue(result);
+
+            global.beforeEachStart = jest.fn();
+
+            result.context.driver.run = jest.fn();
+            result.context.driver.newPage = jest.fn();
+            result.context.driver.goto = jest.fn();
+
+            await execCase(context);
+
+            expect(result.context.driver.run).not.toBeCalled();
+
+            context.driver = "puppeteer";
+
+        });
+
+        it('execCase, verify the params of global and global.beforeEachStart should be called', async () => {
+
+            context.isHeadless = true;
+            context.isDebugger = true;
+            context.isVerbose = true;
+
+            result = await getExecCaseParams(context);
+
+            global.getExecCaseParams = jest.fn().mockReturnValue(result);
+
+            global.beforeEachStart = jest.fn();
+
+            result.context.driver.run = jest.fn();
+            result.context.driver.newPage = jest.fn();
+            result.context.driver.goto = jest.fn();
+
+            await execCase(context);
+
+            expect(result.context.driver.run).toBeCalled();
+
+        });
+
+
+        it('execCase, if isSandbox is false context.driver.run should not be called', async () => {
+
+            context.isSandbox = false;
+
+            const result = await getExecCaseParams(context);
+
+            global.getExecCaseParams = jest.fn().mockReturnValue(result);
+
+            global.beforeEachStart = jest.fn();
+
+            result.context.driver.run = jest.fn();
+            result.context.driver.newPage = jest.fn();
+            result.context.driver.goto = jest.fn();
+
+            await execCase(context);
+
+            expect(result.context.driver.run).not.toBeCalled();
+
+        });
+
+    }),
+
+    describe('postSetup unit test : testCase', () => {
```

Comment:
Need test global.execDrivers include several drivers(puppeteer,chrome...),then check the number of calls to execCase.
Similarly, you need to consider other parameters，for example execTags,groups and options